03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash +x
```

Comment:
Is this option (-x) required in release version?

Reply:
It's mistake.